.ci/scripts/download-png-from-kibana.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+kibana_host=$1
+kibana_user=$2
+kibana_pwd=$3
+png_output_file=${4:-'out.png'}
+table_width_px=${5:-850}
+
+png_url_path=$(curl -XPOST --silent -L -u "$kibana_user:$kibana_pwd" -H 'kbn-xsrf: true' "$kibana_host/api/reporting/generate/pngV2?jobParams=%28browserTimezone%3AEurope%2FBerlin%2Clayout%3A%28dimensions%3A%28height%3A0%2Cwidth%3A$table_width_px%29%2Cid%3Apreserve_layout%29%2ClocatorParams%3A%28id%3ADASHBOARD_APP_LOCATOR%2Cparams%3A%28dashboardId%3Aa5bc8390-2f8e-11ed-a369-052d8245fa04%2CpreserveSavedFilters%3A%21t%2CtimeRange%3A%28from%3Anow-30d%2Cto%3Anow%29%2CuseHash%3A%21f%2CviewMode%3Aview%29%2Cversion%3A%278.3.2%27%29%2CobjectType%3Adashboard%2Ctitle%3Aapp_bench_diff_shifts_slack%2Cversion%3A%278.3.2%27%29" \
```

Comment:
It looks like the previous dashboard has been modified.

Where we had markdown objects with detailed explanations on what to look for before, they are now gone. What happened? Would you mind bringing those back?

Reply:
The idea is to grab the data from the existing table. It allows us to add/modify any metrics to comparison without changing any code. 
However it turned our that CSV reporting is now broken in Kibana. As soon as it fixed I going to use csv to create detailed comparison with markdown. 
Temporarily use the png image to share the results. It is less representative but the report can be adjusted on the fly.